interact/src/lib.rs

```diff
@@ -21,13 +21,20 @@ pub struct Config {
     pub denom: String,
     pub mnemonic: String,
     pub password: String,
+    pub account_prefix: String,
+    pub account_address: String,
 }
 
 impl Config {
+    pub fn set_env(key: &str, value: &str) {
```

Comment:
it would be better to implement `read_from_file` or `read_from_path` to read config file directly from the given file path.